docs/topics/constructing-collections.md

```diff
@@ -31,6 +31,24 @@ initialization fluent here.
 val numbersMap = mutableMapOf<String, String>().apply { this["one"] = "1"; this["two"] = "2" }
 ```
 
+## Create with collection builder functions
+
+Another way of creating a collection is to call a builder function –
+[`buildList()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/build-list.html), [`buildSet()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/build-set.html),
+or [`buildMap()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/build-map.html). They create a new
+mutable collection of the corresponding type, populate it using [write operations](collection-write.md),
+and return a read-only collection with the same elements.
```

Comment:
It's better to use a colon if the following code block shows an example.

Reply:
Thanks, fixed